server/src/main/java/org/elasticsearch/cluster/SnapshotDeletionsInProgress.java

```diff
@@ -176,17 +231,23 @@ public String toString() {
     public static final class Entry implements Writeable, RepositoryOperation {
         private final List<SnapshotId> snapshots;
         private final String repoName;
+        private final State state;
         private final long startTime;
         private final long repositoryStateId;
+        private final String uuid;
```

Comment:
Needed the new id here to get some fixed coordinate/key for a delete operation to connect completion listeners to when batching deletes up.